broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -57,9 +62,11 @@
   private final String name;
   private final RetryStrategy exportingRetryStrategy;
   private final RetryStrategy recordWrapStrategy;
+  private final List<FailureListener> listeners = new ArrayList<>();
   private LogStreamReader logStreamReader;
   private EventFilter eventFilter;
   private ExportersState state;
+  private HealthStatus healthStatus = HealthStatus.HEALTHY;
```

Comment:
```suggestion
  private volatile HealthStatus healthStatus = HealthStatus.HEALTHY;
```

`healthStatus` is not accessed via usual actor methods. Since it can be accessed by concurrent threads, it should be volatile.